app/controllers/transaction_controller.rb

```diff
@@ -2,6 +2,8 @@ class TransactionController < ContentItemsController
   include Cacheable
   include LocaleHelper
 
+  force_html_format
```

Comment:
This is how it will look like in controllers. It can also be added to the parent controller.